.editorconfig

```diff
@@ -12,6 +12,10 @@ max_line_length = 233
 indent_style = space
 indent_size = 2
 
+[.prettierrc]
```

Comment:
since they are all the same it could be: 
```
[{*.json,*.yml,.prettierrc}]
indent_style = space
indent_size = 2
```

Reply:
Using `.prettierrc.js` or `.prettierrc.json` would be better. Nowadays, many projects supports/requires an extension (eslint, babel, …)